CHANGELOG.md

```diff
@@ -14,6 +14,11 @@
 
 ### What's New
 
+- A new **Ruby** codegen backend has been added. You can now call `uniffi-bindgen -l ruby` to
+  generate a Ruby module that wraps a UniFFI Rust component. Thanks to @saks for contributing
+  this backend!
+    - When running `cargo test` locally, you may need to install Ruby and the `ffi` gem
```

Comment:
some time later we should try and improve this a little - It can be tiresome to filter through test failure output each time and go "yep, that just the lack of swift failing" etc - especially when running from different environments, each of which has a different set of test deps installed.

Reply:
Yeah, probably it makes more sense to log a message about skipping the test than the fail it; filed https://github.com/mozilla/uniffi-rs/issues/466 for followup.